SCons/Tool/MSCommon/vc.py

```diff
@@ -735,21 +911,35 @@ def reset_installed_vcs():
 # of time not parsing every time.
 
 script_env_cache = None
+re_output_error = None
 
 def script_env(script, args=None):
     global script_env_cache
+    global re_output_error
 
     if script_env_cache is None:
         script_env_cache = common.read_script_env_cache()
     cache_key = "{}--{}".format(script, args)
     cache_data = script_env_cache.get(cache_key, None)
     if cache_data is None:
+        if re_output_error is None:
+            re_output_error = re.compile('^(' + '|'.join([
```

Comment:
Can initialize this at top of file?

Reply:
Yes it can.  Will move.

Reply:
use a raw string for regexes for general consistency

Reply:
Will do.